eureka-client/src/main/java/com/netflix/discovery/CacheRefreshedEvent.java

```diff
@@ -6,6 +6,9 @@
  * 
  * @author brenuart
  */
-public class CacheRefreshedEvent extends DiscoveryEvent {
-
+public class CacheRefreshedEvent extends AbstractEurekaEvent {
```

Comment:
Agree here. It is preferable to run with the two different event mechanisms side-by-side for now, until future client refactors.